docs/REFERENCE-VALIDATION.md

```diff
@@ -0,0 +1,18 @@
+# Remove reference validation
+
+If a user wants to remove referential integrity checking to facilitate faster loading of test data, the following database scripts can be used to remove the foreign key constraints on the tables.
+
+## Getting the list of foreign key constraints on the table
+
+``` sql
+SELECT conname
+FROM pg_constraint
+WHERE conrelid = '<table-name>'::regclass
+AND contype = 'f';
+```
+
+## Remove reference validation constraint from `References` table
+
+``` sql
+ALTER TABLE "references"  DROP CONSTRAINT FK_References_ReferencedAlias
```

Comment:
What about the `aliases` table?

Reply:
As discussed with @bradbanister, this is the only constraint that will validate the references while inserting the documents. https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/pull/152/files#diff-2a9b535d96d2ca7dcb9a4e1076a34396ee6cf48f9409b757eb2f2d27d005a9e8

Reply:
In that case, it would be helpful to tell the reader that they should _not_ delete the `aliases` foreign key.